README.md

```diff
@@ -37,7 +37,7 @@ Build a sample vulnerable program:
 ```bash
 $ cd example
 $ make sf
-clang-sf -fsanitize=address -O1 main.c -c -o main.sf.o
+clang-sf -fsanitize=address -O1 demo.c -c -o main.sf.o
```

Comment:
This change is already present in another pull request you'd made. No need to add it here as well.

Reply:
Whoops this and the other doc change is due to my own bad skill with Github PRs. I'll clean this up.